cli_reference/basic_cli_operations.adoc

```diff
@@ -295,10 +295,6 @@ indicated files.
 |`oc export _<object_type>_ [--options]`
 |Export resources to be used elsewhere
 
-|`groups`
-|`oc groups [--options]`
-|Create or modify groups.
-
```

Comment:
Since we don't have a dedicated admin CLI topic (we should make one), just removing this from this developer CLI topic for now.